### DISCUSSION TITLE
Unable to pull away code

### DISCUSSION THREAD

**Discussion by aimuz** [community/community#36097] [Pull Requests]
https://github.com/kubernetes/kubernetes/pull/112675

In this PR, there is a problem with not being able to pull the code

In the CI implementation, the following actions were performed and it was not possible to get

```
W1014 03:30:05.495] git fetch failed
I1014 03:30:07.098] Call:  git fetch --quiet --tags https://github.com/kubernetes/kubernetes master +refs/pull/112675/head:refs/pr/112675
W1014 03:30:07.769] fatal: Couldn't find remote ref refs/pull/112675/head
```

I tried to execute the following command locally and had the same failure

```
gh pr checkout 112675
fatal: Couldn't find remote ref refs/pull/112675/head
```

**Comment by Khushboqof** [comment]
You can link a pull request or branch to an issue to show that a fix is in progress ... no links are created and merging the PR has no effect on the issues.

**Comment by wuhuizuo** [comment]
Happened again!

**Reply by wuhuizuo** [reply]
1. Get PR merge status failed: ![img_v3_02h0_e27e3a4b-f307-4b16-9a61-f2932ff3e89g](https://github.com/user-attachments/assets/fb174033-9f6d-43d0-8d21-d00397698675).
2. Fetch head for PRs failed, one of them: https://github.com/pingcap/tidb/pull/57712/ 
```
HEAD is now at b11d034100 ttl: enlarge the max split of TTL job (#57701)

POST git-upload-pack (656 bytes)

fatal: couldn't find remote ref refs/pull/57712/head

script returned exit code 128
```

**Comment by purelind** [comment]
We also encountered this issue in the https://github.com/pingcap/tidb and https://github.com/pingcap/tiflow
* pr  https://github.com/pingcap/tidb/pull/57712
 ```shell
gh pr checkout 57712
fatal: couldn't find remote ref refs/pull/57712/head
failed to run git: exit status 128
 ```
* pr https://github.com/pingcap/tidb/pull/57706
```
gh pr checkout succeeded but did not pull the latest commit.
```
